etna/core/models/mixins.py

```diff
@@ -208,3 +214,158 @@ class Meta:
     api_fields = [
         APIField("page_sidebar"),
     ]
+
+
+class SocialMixin(models.Model):
+    """Mixin to add social media sharing options to a Page."""
+
+    teaser_text = models.TextField(
+        verbose_name=_("teaser text"),
+        help_text=_(
+            "A short, enticing description of this page. This will appear in promos and under thumbnails around the site."
+        ),
+        max_length=160,
+    )
+    teaser_image = models.ForeignKey(

```

Comment:
The teaser text and teaser images aren't specifically for sharing on social media. We use them within other pages in the site.

Could we move `teaser_text` and `teaser_image` to the `base_page` so all pages have them? This would avoid the possibility of needing text and a teaser image and a page not having one.

Reply:
The `SocialMixin` is inherited by `BasePage`, but I can put `teaser_image` and `text` back on just `BasePage`?

Reply:
I would personally feel better about them being defined explicitly in `BasePage` but I'm going to go with whatever you think 😊